source/direct_debit/index.html.md.erb

```diff
@@ -322,8 +341,6 @@ paying users receive their mandate reference number.
 
 ## Refunds
 
-[Contact us](/support_contact_and_more_information/#contact-us) if you need to issue refunds for any kind of Direct Debit payment.
-
-## Indemnity claims
+If a user asks you for a refund, you can process the refund by signing into your GoCardless account through the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
 
-[Contact us](/support_contact_and_more_information/#contact-us) if you need to handle indemnity claims for any kind of Direct Debit payment.
+If the payment's status is `indemnityclaim`, you do not need to do anything. Your user's bank will refund the payment.
```

Comment:
actually, the service might want to contact the customer to find out why they've created an indemnity claim - we might want to point them to this guidance https://support.gocardless.com/hc/en-gb/articles/115002875285-Chargebacks-Customer-Claims#disputing_a_chargeback. it looks like there's no process for disputing indemnity claims inside of GC admin tool though

Reply:
Added in https://github.com/alphagov/pay-tech-docs/pull/306/commits/7ad6f07c2dc6e261a7724b1f8e665c2180950b91.